src/main/java/lotto/view/OutputView.java

```diff
@@ -25,17 +29,33 @@ public static void printLottoNumbers(List<LottoNumber> numbers) {
         System.out.println(numbers);
     }
 
-    public static void printWinningStatics(List<WinningInformationDto> informationList) {
+    public static void printLottoNumbers(LottoTickets lottoTickets) {
+        List<LottoTicket> lottoTicketList = lottoTickets.getLottoTickets();
+        for (LottoTicket lottoTicket : lottoTicketList) {
+            OutputView.printLottoNumbers(lottoTicket.getLottoNumbers());
+        }
+    }
+
+    public static void printWinningStatics(Map<LottoRank, Integer> map) {
         System.out.println(WINNING_STATICS);
         System.out.println(DIVIDING_LINE);
-        for (WinningInformationDto information : informationList) {
-            String format = String.format(WINNING_STATICS_FORMAT, information.getNumberOfMatching(), information.getPrizeMoney(), information.getNumberOfLotto());
-            System.out.println(format);
-
+        List<LottoRank> lottoRankList = Arrays.asList(LottoRank.FIRST, LottoRank.SECOND, LottoRank.THIRD, LottoRank.FOURTH, LottoRank.FIFTH);
+        for (LottoRank lottoRank : lottoRankList) {
+            String format = getWinningStaticsFormat(lottoRank);
+            String formattedString = String.format(format, lottoRank.countOfMatch(), lottoRank.winningPrize(), map.get(lottoRank));
+            System.out.println(formattedString);
         }
+
         System.out.println();
     }
 
+    private static String getWinningStaticsFormat(LottoRank lottoRank) {
+        if (lottoRank == LottoRank.SECOND) {
```

Comment:
추후에 메시지를 보내는 방식으로 연습해보시는 것도 좋을 것 같습니다.
```suggestion
        if (lottoRank.isSecond()) {
```